config/default.conf

```diff
@@ -6,7 +6,7 @@ server {
 }
 
 server {
-	server_name ckan.arbidata.com;
+	server_name cct.opencitieslab.org;

```

Comment:
I think we might have to look at some templating for this file, and the `production.ini` one - you can do some rudimentary stuff with SED, or go a little fancier with a jinja based setup.

Any preference?